sale_outplacement/models/project.py

```diff
@@ -38,6 +47,16 @@ class ProjectTask(models.Model):
         string="Task Type",
     )
 
+    @api.onchange('start_date', 'end_date')
+    def _compute_duration(self):
+        if self.start_date and self.end_date:
+            diff = self.end_date - self.start_date
+            days = diff.days*24
```

Comment:
days = diff.days*24 would be the amount of hours in the diff and not the amount of days

A better solution would be:
hours = round(diff.total_seconds() / 3600, 2)
timedelta.total_seconds() is the total amount of seconds in the diff, whilst seconds is the remainder after days are calculated.
So whilst your calculations seems to be correct they are a bit confusing and could improve.